docs/getting-started/my-account/README.md

```diff
@@ -91,36 +91,25 @@ You can select the order of the comments (for example of the comments for a work
 
 If you choose newest first the latest comment will appear on top in the Activity list.
 
-![order-to-display-comments](order-to-display-comments.png)
+![my-account-settings-comments](order-to-display-comments.png)
```

Comment:
please rename the image file and change the name here, so there are no spaces in it.